src/test/regress/expected/isolation_citus_dist_activity.out

```diff
@@ -232,10 +232,10 @@ step s1-begin:
     BEGIN;
 
 step s2-begin:
-	BEGIN;
+ BEGIN;
```

Comment:
nothing actionable for this PR. but this white space changes in general that almost always happens on isolation test outputs are very annoying. Do you know why we end up so often/many changes almost always?

Reply:
I have a guess and a partial fix that will IMO make these space changes less frequent in the future: https://github.com/citusdata/citus/pull/5981